src/App.css

```diff
@@ -0,0 +1,14 @@
+#root {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  max-width: 1200px;
+  margin: 0 auto;
+  text-align: center;
+  width: 100%;
+}
+
+.card {
```

Comment:
I would recommend to avoid global styles especially like that class

Reply:
Yes

Reply:
Done